docs/changelog.rst

```diff
@@ -3,6 +3,19 @@
 Changelog
 =========
 
+Version 3.2.0
+-------------
+
+* `PR-256 <https://github.com/confluentinc/kafka-rest/pull/256>`_ - Update ClusterTestHarness to use o.a.k.common.utils.Time.
+* `PR-255 <https://github.com/confluentinc/kafka-rest/pull/255>`_ - clarified the key/value format limitations based on user questions
+* `PR-264 <https://github.com/confluentinc/kafka-rest/pull/264>`_ - Fix build to work post KIP-103
```

Comment:
I guess this changelog is auto generated, but there is no point in reference interim commits (fixes of fixes) in changelogs.